src/pages/Main/Favorites.jsx

```diff
@@ -0,0 +1,114 @@
+import useAuth from "../../auth/useAuth";
+import React, {useEffect, useState} from "react";
+import {useNavigate} from "react-router-dom";
+import {
+    backendApiCall,
+    fetchFavorites, fetchFavoritesAndAddUserQuizzes,
+    getFavorites,
+    removeFavorite
+} from "../../functions/exportFunctions";
+import {LOGIN} from "../../App";
+import {QuizzesContainer} from "./Main";
+
+export const Favorites = ({
```

Comment:
This happens because there is no user progress functionality on the favorites branch on the backend, you need to merge the dev branch into favorites to make it works.